package.json

```diff
@@ -81,6 +81,7 @@
 		"build-client-both": "CONCURRENT_BUILDS=2 concurrently -c cyan -n \"fallback ,evergreen\" \"npm run build-client-fallback\" \"npm run build-client-evergreen\"",
 		"build-client-if-prod": "node -e \"process.env.CALYPSO_ENV === 'production' && process.exit(1)\" || npm run build-client-both",
 		"build-client-if-desktop": "node -e \"(process.env.CALYPSO_ENV === 'desktop' || process.env.CALYPSO_ENV === 'desktop-development') && process.exit(1)\" || npm run build-client-fallback",
+		"build-client-stats": "cross-env-shell NODE_ENV=production CALYPSO_ENV=production EMIT_STATS=true PROGRESS=true npm run build-client-evergreen",
```

Comment:
If we have issues with `cross-env-shell`, I think we can drop it here.

Reply:
We can drop `cross-env` completely, but it means dropping the remaining support for Windows that we have.

`cross-env` doesn't do anything useful on Unix systems: the shell already understands the `ENV_VAR=value command --to-run` syntax. It's useful for Windows: there, `cross-env` will parse the `ENV_VAR=value` prefixes, and sets up the environment and runs `command --to-run` in a way that Windows `cmd.exe` understands.

While removing `cross-env`, we could also get rid of `rimraf` and friends and replace them with `rm -rf`. That saves time where `npm rimraf` needs to download the package from the NPM registry.

Reply:
+1. I think it's reasonable to remove the ability for non-Unix developers to generate stats while we wait for the `cross-env-shell` fix.

Reply:
> We can drop `cross-env` completely, but it means dropping the remaining support for Windows that we have.

I'm not sure I'd remove it permanently, more as a temporary measure. I'm hoping that `cross-env-shell` will be fixed at some point, or perhaps we can find an earlier version of it without the bug.

Reply:
OK, I removed the `cross-env` usages in https://github.com/Automattic/wp-calypso/pull/39502/commits/02d3d34c9aff26e8c0bf713ab271e7cd9106226b

The `terser` issue and the swallowed exit codes can bite at any time, including a production build.